src/main/java/Duke.java

```diff
@@ -1,10 +1,42 @@
+import duke.*;
```

Comment:
Perhaps import each class explicitly, as per the Java coding standard. I noticed the same issue in several other places too.